src/components/Toolbar.js

```diff
@@ -97,7 +144,7 @@ const Toolbar = ({ onDownloadCropClick, onDownloadFullSize }) => {
         id="previous"
         data-testid="previous"
         href="#previous"
-        className="toolbar-controls button-link"
+        css={toolbarControl}
         title="Previous"
       >
         <FontAwesomeIcon icon="arrow-circle-left" />
```

Comment:
- Navigating through images using `next` icon does not update   `active ` thumbnail image. 
- Download also does not update to `active` image.

Reply:
@katdivyareddy10 Got 'em.  There were a few bugs that we must not have ported over from fixes in Northwestern's Digital Collections viewer.  This should be wired up bug free now...